lib/crypto/butil.js

```diff
@@ -35,7 +35,7 @@ module.exports.xorBuffers = function xorBuffers(buffer1, buffer2) {
 module.exports.unbuffer = function unbuffer(object, inplace) {
   var keys = Object.keys(object)
   var copy = inplace ? object : {}
-  for (var i = 0; i < keys.length; i++) {
+  for (var i = 0; i < keys.length; i ++) {
```

Comment:
@vladikoff: I _think_ the ESLint rule you're looking for is [`space-unary-ops`](http://eslint.org/docs/rules/space-unary-ops); plus you'll need some overrides for `!` and `!!`:

```yaml
env:
  es6: true
  node: true

extends:
  - eslint:recommended

root: true

rules:
  no-console: off
  space-unary-ops: [error, {words: true, nonwords: false, overrides: {'!': true, '!!': true}}]
```

That should allow the following rough styles:

```js
function foo(a, b) {
  a = a++;
  b = ++b;
  return !! a || !! b;
}

const res = foo(3, 4);
console.log(res);
```

If you're lucky,  you can run `eslint . --fix` and have it magically autofix+format all the code for you, but that may cause a slightly insane diff, depending on current config.